Bio/PDB/Atom.py

```diff
@@ -387,31 +387,31 @@ def detach_parent(self):
         """Remove reference to parent."""
         self.parent = None
 
-    def get_sigatm(self):
+    def get_sigatm(self) -> Optional[np.ndarray]:
         """Return standard deviation of atomic parameters."""
         return self.sigatm_array
 
-    def get_siguij(self):
+    def get_siguij(self) -> Optional[np.ndarray]:
         """Return standard deviations of anisotropic temperature factors."""
         return self.siguij_array
 
-    def get_anisou(self):
+    def get_anisou(self) -> Optional[np.ndarray]:
         """Return anisotropic B factor."""
         return self.anisou_array
 
-    def get_parent(self):
+    def get_parent(self) -> Optional[Residue]:
```

Comment:
This is breaking in the tests, ``NameError: name 'Residue' is not defined``

I guess either add the import, or try string based type annotation?